app.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topics");
+const app = express();
+app.use(express.json());
```

Comment:
Good thinking in that you will use this later. But Does it need to be there now?